### ISSUE TITLE
tiled not working as it should?

### ISSUE THREAD

**Issue by ryltenius** [TomAnthony/itermocil#20]
Hi,

I do not think the commando tiled work as it should or I am miss understand the function.

If I try the example:

windows:
  - name: _tiled_4_panes
    root: ~
    layout: tiled
    panes:
      - echo "pane 1"
      - echo "pane 2"
      - echo "pane 3"
      - echo "pane 4"

I get this result:
![image](https://cloud.githubusercontent.com/assets/5538384/10967154/d5b137dc-83b8-11e5-8e99-6b260ac4cdd0.png)

It should be like this ?

![image](https://cloud.githubusercontent.com/assets/5538384/10967180/f4335992-83b8-11e5-86eb-75ff57ae26fa.png)

Specs:
iTerm2 build 2.1.4
brew list itermocil
/usr/local/Cellar/itermocil/0.1.7/bin/itermocil
/usr/local/Cellar/itermocil/0.1.7/bin/itermocil.py
/usr/local/Cellar/itermocil/0.1.7/libexec/bin/ (3 files)
/usr/local/Cellar/itermocil/0.1.7/libexec/vendor/ (38 files)

**Comment by tobsn** [comment]
same here!

**Comment by TomAnthony** [comment]
I'm unable to assign this.

@ryltenius - if possible can you screenshot the 'General' and 'Appearance' tabs of your iTerm preferences? Do other layouts work as expected?

**Comment by Kamii** [comment]
As i understand it after reading the code cmd+] is not working. So when this command is trying to be executed `self.applescript.append(prefix + 'keystroke "]" using command down')` it will not rotate the selected pane. You need to bind cmd+] to the action "next pane". 

I hade some difficulties with this. Navigate to next window is binded to cmd+alt+number and therefor i couldn't bind cmd+] (on a swedish keyboard alt+9 will result in ]) to next pane.

TL;DR on a Swedish keyboard the next pane command is not working. It tries to switch window to window nr 9.

**Comment by Kamii** [comment]
Downloaded iTerm2 2.9.20160206 beta (OS 10.8+) and tiled layout is working.

**Comment by TomAnthony** [comment]
Hrm - that sucks, but I'm unsure how to address. Also given iTerm 3.0.0 is almost ready I'm not super worried about trying to find (more) hacks for the old version (where everything is a bit fiddly already). 

Thanks a lot for the working out the source of the problem. :)
